core/src/layers/prometheus.rs

```diff
@@ -134,6 +136,15 @@ impl PrometheusLayer {
         }
         self
     }
+
+    /// set path label level
+    /// 0: no path label, the path label will be the ""
+    /// >0: the path label will be the path split by "/" and get the last n level, like "/abc/def/ghi", if n=1, the path label will be "/abc"
+    /// -1: the path label will be full path
+    pub fn path_label_level(mut self, level: i32) -> Self {
+        self.path_label_level = level;
```

Comment:
How about adding API like `enable_path_label(mut self, level: usize)` while:

- `0` means we don't record `path`, which is the default value.
- `lvl` means the level we will record.

We should not provide full path support (users can pass `usize::MAX` if they want).